### PR TITLE
Add V2C number description

### PR DESCRIPTION
**Pull request by dgomes**
## Proposed change

Adds sensor description

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103681
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by edenhaus — DISMISSED** [review]
Please update also `ha_platforms` on line 12

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @dgomes :+1:

File: source/_integrations/v2c.markdown [hunk 1]
```diff
@@ -46,6 +47,12 @@ The V2C integration currently exposes the following sensors:
 - house_power: Power consumed by your household. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle supply equipment">EVSE</abbr>).
 - fv_power: Power produced by your photovoltaic system. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle supply equipment">EVSE</abbr>).
 
+## Number
+
+The V2C integration currently exposes the following number entity:
+
+- intensity: current used to transfer energy to your vehicle
```
Comment (c0ffeeca7) [inline source/_integrations/v2c.markdown]:
```suggestion
- intensity: current (in A) used to transfer the energy to your vehicle
```

**Review by c0ffeeca7 — DISMISSED** [review]

File: source/_integrations/v2c.markdown [hunk 2]
```diff
@@ -46,6 +47,12 @@ The V2C integration currently exposes the following sensors:
 - house_power: Power consumed by your household. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle supply equipment">EVSE</abbr>).
 - fv_power: Power produced by your photovoltaic system. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle supply equipment">EVSE</abbr>).
 
+## Number
+
+The V2C integration currently exposes the following number entity:
+
+- intensity: current (in A) used to transfer the energy to your vehicle
```
Comment (c0ffeeca7) [inline source/_integrations/v2c.markdown]:
```suggestion
- intensity: current used to charge your vehicle
```

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**